etc/library.sh

```diff
@@ -114,51 +119,151 @@ function run_app_unsafe()
   chown root:www-data $log
 
   echo "Running $ncp_app"
-  echo "[ $ncp_app ]" >> $log
-
-  # read script
-  unset configure
-  source "$script"
 
-  # read cfg parameters
-  [[ -f "$cfg_file" ]] && {
-    local cfg="$( cat "$cfg_file" )"
-    local len="$(jq '.params | length' <<<"$cfg")"
-    for (( i = 0 ; i < len ; i++ )); do
-      local var="$(jq -r ".params[$i].id"    <<<"$cfg")"
-      local val="$(jq -r ".params[$i].value" <<<"$cfg")"
-      eval "$var=$val"
-    done
+  # Check if app is already running in tmux
+  running_app=$( [[ -f "$LOCK_FILE" ]] && cat "$LOCK_FILE" || echo "" )
+  [[ ! -z $running_app ]] && which tmux > /dev/null && tmux has-session -t="$running_app" > /dev/null 2>&1 && {
+    #echo "Already running. Attaching to output..." | tee -a $log
+    
+    local choice
+    [[ $ATTACH_TO_RUNNING == "1" ]] && choice="y"
+    [[ $ATTACH_TO_RUNNING == "0" ]] && choice="n"
+    question="An app ($running_app) is already running. Do you want to attach to it's output? <y/n>"
+    if [[ $choice == "y" ]] || [[ $choice == "n" ]]
+    then
+      echo "$question"
+      echo "Choice: <y>"
```

Comment:
```suggestion
      echo "Choice: <$choice>"
```

Reply:
Totally missed that :D

Reply:
Fixed in 6c8e95c4d79ac6ebae5983397fbe4887052a9986